src/eventProcessor.ts

```diff
@@ -158,6 +159,21 @@ export default async (event: ContractEvent): Promise<void> => {
       return;
     }
 
+    case ContractEventsSignatures.ColonyRoleSet: {
```

Comment:
This could read: 
``` js
case ContractEventsSignatures.ColonyRoleSet: 
case ContractEventsSignatures.ColonyRoleSet_OLD:
case ContractEventsSignatures.RecoveryRoleSet: {
 await handleManagePermissionsAction(event);
      return;
}
```